data/distro.yml

```diff
@@ -564,30 +564,41 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
-- name: "SulinOS"
-  logo: "sulinos.svg"
+- name: "TurkMan Linux"
+  logo: "turkman.svg"
```

Comment:
It seems to be called _Turkman Linux_, not _Turk**M**an Linux_.

Reply:
fixed